tests/networkpolicy_test.go

```diff
@@ -116,15 +129,18 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 		AfterEach(func() {
 			Expect(virtClient.NetworkingV1().NetworkPolicies(vmia.Namespace).Delete("deny-by-default", &v13.DeleteOptions{})).To(Succeed())
```

Comment:
This also needs fixing, it is incorrect to delete something that was not yet created (e.g. if something failed before a creation occurred). True for all the rest as well.